libs/dal/src/repositories/message/message.schema.ts

```diff
@@ -132,6 +127,17 @@ messageSchema.virtual('template', {
 
 messageSchema.plugin(mongooseDelete, { deletedAt: true, deletedBy: true, overrideMethods: 'all' });
 
+messageSchema.index({
+  _notificationId: 1,
+  transactionId: 1,
+  _subscriberId: 1,
+  _messageTemplateId: 1,
+  _templateId: 1,
+  identifier: 1,
```

Comment:
I am not entirely regarding the 'identifier' i think we need to removed it, added it here so we can talk about it in the PR. this one is used in order to filter the `identifier` of the feedId, because it is used on the widget route all the time i was not sure if to keep it or not.